kobo/settings/base.py

```diff
@@ -750,3 +757,31 @@ def __init__(self, *args, **kwargs):
     'medium': '_medium',
     'small': '_small',
 }
+
+TRENCH_AUTH = {
+    'USER_MFA_MODEL': 'trench.MFAMethod',
+    'USER_ACTIVE_FIELD': 'is_active',
+    'BACKUP_CODES_QUANTITY': 5,
+    'BACKUP_CODES_LENGTH': 12,  # keep (quantity * length) under 200
+    'BACKUP_CODES_CHARACTERS': (string.ascii_letters + string.digits),
+    'DEFAULT_VALIDITY_PERIOD': 30,
```

Comment:
Answer: we could make it configurable, but changing the value would break TOTP for *everyone* who has already set it up. Also, 30 seconds seems to be the most common validity period in general use